components/Button/Button.tsx

```diff
@@ -0,0 +1,39 @@
+import clsx from 'clsx';
+import React, { ButtonHTMLAttributes, FC, PropsWithChildren } from 'react';
+
+export const enum ButtonVariant {
+  SOLID = 'solid',
+  DASHED = 'dashed',
+}
+
+interface Props extends PropsWithChildren<ButtonHTMLAttributes<HTMLButtonElement>> {
+  variant?: ButtonVariant;
+  className?: string;
+}
+
+export const Button: FC<Props> = ({
+  children,
+  variant = ButtonVariant.SOLID,
+  className,
+  ...props
+}) => {
+  const isDashedBtn = variant === ButtonVariant.DASHED;
+
+  return (
+    <button
+      className={clsx(
+        'rounded font-medium',
+        {
+          [clsx(
+            'border border-dashed border-gray-500 px-2 py-1',
+            'text-sm leading-normal text-gray-500',
+          )]: isDashedBtn,
+        },
+        className,
```

Comment:
```suggestion
{
 'border border-dashed border-gray-500 px-2 py-1' : isDashedBtn,
'text-sm leading-normal text-gray-500' : isDashedBtn
}
```

Reply:
I think writing this way might mislead that these two lines belong to 2 different conditions while it's actually one